packages/ui/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ The following is a curated list of changes in the Enact ui module, newest change
 
 - `ui/Marquee.MarqueeController` to start animation properly when `marqueeOnFocus` is set to `true` and text changed
 - `ui/Scroller` and `ui/VirtualList` to have default prop when `undefined` prop is passed
+- `ui/VirtualList.VirtualListBasic` to update its scroll bounds when the total width of items is changed
```

Comment:
```suggestion
- `ui/VirtualList` to update its scroll bounds when the total width of items is changed
```

Reply:
Could you also change `ui/VirtualList.VirtualListBasic` to `ui/VirtualList` at line 9?
It's enough to say `ui/VirtualList`.

Reply:
🤔 Is `total width of items` right term?

Reply:
How about 
"to update its scroll bounds when the total width of all items changes"

Reply:
`total size of items` would be better :)

Reply:
It's fixed.